airflow/providers/mongo/sensors/mongo.py

```diff
@@ -41,13 +43,18 @@ class MongoSensor(BaseSensorOperator):
     template_fields = ('collection', 'query')
 
     @apply_defaults
-    def __init__(self, collection, query, mongo_conn_id="mongo_default", *args, **kwargs):
+    def __init__(self,
+                 collection: str,
+                 query: dict,
+                 mongo_conn_id: str = "mongo_default",
+                 *args,
+                 **kwargs) -> None:
         super().__init__(*args, **kwargs)
         self.mongo_conn_id = mongo_conn_id
         self.collection = collection
         self.query = query
 
-    def poke(self, context):
+    def poke(self, context: Optional[dict]) -> bool:
```

Comment:
```suggestion
    def poke(self, context: dict) -> bool:
```
It's a required parameter.